fdbcli/DebugCommands.actor.cpp

```diff
@@ -395,8 +397,35 @@ ACTOR Future<bool> checkallCommandActor(Database cx, std::vector<StringRef> toke
 		}
 		wait(delay(1.0));
 	}
+	return consistent;
+}
 
-	printf("Checking complete.\n");
+// The command is used to check the inconsistency in a keyspace, default is \xff\x02/blog/ keyspace.
```

Comment:
We can remove ", default is \xff\x02/blog/ keyspace" now.

Reply:
Yes.